### PR TITLE
[golang] update for Go 1.7.4

### PR DESCRIPTION
**Pull request by joelhandwell**
If google brought you here and you want choco install now, do following:

```bat
git clone https://github.com/joelhandwell/chocolatey-golang.git --branch 1_7_4
cd chocolatey-golang\golang
cpack
choco uninstall golang
choco install -y golang.1.7.4.nupkg
```

### CODE REVIEW

**Comment by joelhandwell** [comment]
@carstn is there anything I can help to merge this?

**Comment by sschuberth** [comment]
I'd also like to see this merged, preferably before [Go 1.8](https://groups.google.com/forum/#!topic/golang-nuts/tr2ZKSQ42zE) gets released ;-)

**Comment by joelhandwell** [comment]
@carstn ping

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+**.nupkg
```
Comment (carstn) [inline .gitignore]: I'd like to keep each version's `.nupkg`s in git.

**Comment by carstn** [comment]
Thanks @joelhandwell. I packed and added the .nupkg, pushed to chocolate.org/golang for moderation, and merged these changes in #11.

File: .gitignore (hunk 1 above)
Comment (joelhandwell) [inline .gitignore]: ok

**Comment by sschuberth** [comment]
Great, thanks all! Any idea how long it will take to actually see the package at https://chocolatey.org/packages/golang?

**Comment by carstn** [comment]
@sschuberth Moderation has taken roughly 3-7 days in the past, so it should be approved within the next few days. 
Pending versions are available right now if you're willing to trust the package (e.g. `choco install golang --version 1.7.5`); they just haven't been officially reviewed yet.

**Pull request merged**